src/software/telemetry/src/alarm.rs

```diff
@@ -0,0 +1,67 @@
+// MakAir
+//
+// Copyright: 2020, Makers For Life
+// License: Public Domain License[derive(Debug, Eq, PartialEq, Hash)]
```

Comment:
What happened here? 😂

Reply:
Ooh, that's where this code went, I was pretty sure I added it to the AlarmCode enum and then I failed to find it :joy: